test/test_wordcloud_cli.py

```diff
@@ -31,22 +30,12 @@ def all_arguments():
     return arguments
 
 
-def test_argument_spec_matches_to_constructor_args():
```

Comment:
This is in fact no longer needed as the MagicMock with `spec` ensures only arguments which actually exists are passed.